.github/workflows/tests.yml

```
@@ -4,7 +4,7 @@
 name: Tests
 
 on:
-  # pull_request:
+  pull_request:
```

Let's wait for 30 minutes to see if this PR fixes the bug. 
Need to apply the suggested change below before merging.
```suggestion
  # pull_request:
```